Makefile

```diff
@@ -22,6 +22,13 @@ CXX_TYPE ?= clang
 -include $(MATH)make/dependencies
 -include $(MATH)make/libraries
 
+## set compiler standard to C++1y to match Stan
```

Comment:
Why are these being added to the STANCFLAGS? Shouldn't it be in CXXFLAGS?

When I run `make` on a model, it already has `-std=c++1y`, I believe being pulled in from the CmdStan makefiles

Reply:
Thanks.  You're right that it should've been `CXXFLAGS`.  But if it's being picked up from CmdStan, then that's even better as it'll keep up when they bump.  So I'll just remove this change.